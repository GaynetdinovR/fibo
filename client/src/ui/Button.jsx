import classNames from "classnames";
import styles from '../styles/Ui.module.sass'
const Button = ({children, className, onClickFn}) => {
	return(
		<button onClick={() => onClickFn()} className={classNames(className, styles.default_button)}>
			{ children }
		</button>
	)
}

export default Button;