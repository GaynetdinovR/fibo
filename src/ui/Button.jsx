import classNames from "classnames";
import styles from '../styles/Ui.module.sass'
const Button = ({children, className}) => {
	return(
		<button className={classNames(className, styles.default_button)}>
			{ children }
		</button>
	)
}

export default Button;