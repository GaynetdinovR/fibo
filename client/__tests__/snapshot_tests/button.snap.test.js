import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import Button from "../../src/ui/Button.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<Button>Button</Button>).toJSON();
	expect(tree).toMatchSnapshot();
});
