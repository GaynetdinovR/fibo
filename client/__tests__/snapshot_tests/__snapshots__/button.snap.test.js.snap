// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="default_button"
  onClick={[Function]}
>
  Button
</button>
`;
